packages/prettier/index.js

```diff
@@ -4,6 +4,6 @@ module.exports = {
   semi: true,
   singleQuote: true,
   bracketSpacing: true,
-  arrowParens: "avoid",
-  trailingComma: "all",
-};
+  arrowParens: 'always',
+  trailingComma: 'all',
+}
```

Comment:
is this missing `;` because of the `semi: false` on `prettier.config.js` previously?

Reply:
Updated, config is now in `prettier.config.js` and we reexport it in `index.js`